functions/__tests__/expressExample/resend/index.test.js

```diff
@@ -10,24 +11,64 @@ testConfig.mockConfig({
     issuer: "issuer",
     jwk_uri: "some uri",
   },
+  api: {
+    api_token: "sometoken",
+    url: "fake_url",
+  },
 });
 
 jest.mock("../../../utils/middleware");
+jest.mock("axios", () => jest.fn());
 
 describe("Supertest", () => {
   afterEach(() => {
     jwtCheck.mockClear();
+    axios.mockClear();
```

Comment:
We are going to need to mock retrieving from the db to get the token as well even though it hasn't been implemented yet.